Source/Logger/trace.cpp

```diff
@@ -129,7 +139,7 @@ void FormatTrace(
 #endif
 
     // [threadId][level][time][area] message
-    auto written = sprintf_s(outputBuffer, "[%04llX][%s][%02d:%02d:%02d.%03u][%s] %s",
+    auto written = _snprintf_s(outputBuffer, SIZE - 3, "[%04llX][%s][%02d:%02d:%02d.%03u][%s] %s",
```

Comment:
why pass SIZE - 3 to a function that ignores the param.  count is commented out above

Reply:
its used for the windows definition.  The definition above is for non-windows only